java/client/src/main/java/glide/api/models/commands/ZmpopOptions.java

```diff
@@ -0,0 +1,19 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.models.commands;
+
+import glide.api.commands.SortedSetBaseCommands;
+
+// TODO add @link to ZMPOP when implemented
+/**
+ * Mandatory option for {@link SortedSetBaseCommands#bzmpop(double, String[], ScoreModifier)} and
+ * for {@link SortedSetBaseCommands#bzmpop(double, String[], ScoreModifier, long)}.
+ */
+public interface ZmpopOptions {
```

Comment:
why not just make this an enum like `ConditionalChange` instead of defining it only for ZMPOP?